helm/tenant/values.yaml

```
@@ -1,189 +1,339 @@
-  # Notice they are in JSON format to be consumed
+  ###
+  # Configure pod logging configuration for the MinIO Tenant.
+  #
+  # - Specify ``json`` for JSON-formatted logs.
+  # - Specify ``anonymous`` for anonymized logs.
+  # - Specify ``quiet`` to supress logging.
+  #
+  # An example of JSON-formatted logs is as follows:
+  #
+  # .. code-block:: shell
+  #
+  #    $ k logs myminio-pool-0-0 -n default
+  #    {"level":"INFO","errKind":"","time":"2022-04-07T21:49:33.740058549Z","message":"All MinIO sub-systems initialized successfully"}
   logging: { }
-  ## serviceMetadata allows passing additional labels and annotations to MinIO and Console specific
-  ## services created by the operator.
+  ###
+  # serviceMetadata allows passing additional labels and annotations to MinIO and Console specific
+  # services created by the operator.
   serviceMetadata: { }
-  ## Add environment variables to be set in MinIO container (https://github.com/minio/minio/tree/master/docs/config)
+  ###
+  # Add environment variables to be set in MinIO container (https://github.com/minio/minio/tree/master/docs/config)
   env: [ ]
-  ## PriorityClassName indicates the Pod priority and hence importance of a Pod relative to other Pods.
-  ## This is applied to MinIO pods only.
-  ## Refer Kubernetes documentation for details https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass/
+  ###
+  # PriorityClassName indicates the Pod priority and hence importance of a Pod relative to other Pods.
+  # This is applied to MinIO pods only.
+  # Refer Kubernetes documentation for details https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass/
   priorityClassName: ""
-  ## additionalVolumes allows adding additional volumes to MinIO pods
+  ###
+  # An array of `Volumes <https://kubernetes.io/docs/concepts/storage/volumes/>`__ which the Operator can mount to Tenant pods.
+  #
+  # The volumes must exist *and* be accessible to the Tenant pods.
   additionalVolumes: [ ]
-  ## additionalVolumeMounts allows mounting additional volumes to MinIO pods
+  ###
+  # An array of volume mount points associated to each Tenant container.
+  # 
+  # Specify each item in the array as follows:
+  #
+  # .. code-block:: yaml
+  #
+  #    volumeMounts:
+  #    - name: volumename
+  #      mountPath: /path/to/mount
+  #
+  # The ``name`` field must correspond to an entry in the ``additionalVolumes`` array.
   additionalVolumeMounts: [ ]
-  ## Define configuration for KES (stateless and distributed key-management system)
-  ## Refer https://github.com/minio/kes
+  # Define configuration for KES (stateless and distributed key-management system)
+  # Refer https://github.com/minio/kes
   #kes:
-  #  ### Image field:
-  #  ## Image from tag (original behaviour), for example:
+  #  ## Image field:
+  #  # Image from tag (original behaviour), for example:
```

Do we care about alternate spellings here?
```suggestion
  #  # Image from tag (original behavior), for example:
```